test/config.test1.yaml

```diff
@@ -20,7 +20,7 @@ countries: ['DE']
 
 snapshots:
   start: "2013-03-01"
-  end: "2014-04-01"
+  end: "2013-03-08"
```

Comment:
that's intended? (guess it makes sense, since we just want to test the code)

Reply:
Yes, shaves about 20 seconds off the runtime.